docs/topics/mta-rn-known-issues-6-2-0.adoc

```diff
@@ -0,0 +1,31 @@
+// Module included in the following assemblies:
+//
+// * docs/release_notes/master.adoc
+
+:_content-type: REFERENCE
+[id="rn-known-issues-6-2-0_{context}"]
+= Known issues
+
+MTA version 6.2.0 has the following issues.
+
+.MTA fails to run an analysis of private GitHub repos
+
+Analysis of private GitHub repositories fails. link:https://issues.redhat.com/browse/MTA-810[MTA-810]
```

Comment:
@anarnold97 looks like it is not a real bug. @sshveta @abrugaro do we have such a case in UI tests?

Reply:
That issue was due to credentials not being associated to the application, so it is not a bug as @mguetta1 said

Reply:
Removed